### PR TITLE
Movie Page API

### PR DESCRIPTION
**Pull request by KrystianGreblowski**
I have added data from API to MoviePage and I have also added many different things to make everything work well.

### CODE REVIEW

File: src/features/MoviePage/Content/MainInfo/styled.js
Comment (Alekzann) [inline src/features/MoviePage/Content/MainInfo/styled.js:1]: Nice decision with grid, as you said previously! Good job!

**Review by Alekzann — CHANGES_REQUESTED** [review]
It was a really great and excellent job! Unfortunately I can't appreciate the work with slice and saga, but the main thing is that it works! 
But, shadow in MoviePage in poster should be bigger, because when we have light image it's hard to see the movie title, crew should includes more persons.
Distance between MainInfo and Poster too big (as for me).

**Comment by KrystianGreblowski** [comment]
Thanks for the review. I changed the distance between MainInfo and Banner. I also changed the number of crew tiles. In my opinion, the box shadow is good enough and can't be bigger because it would take up too much space.

File: src/features/MoviePage/Banner/styled.js [hunk 1]
```diff
@@ -3,30 +3,29 @@ import styled from "styled-components";
...
-export const Shadow = styled.img`
+export const StyledPoster = styled.div`
   max-width: 1356px;
-  height: 100%;
-  z-index: 1;
-  position: absolute;
-  top: 0;
-  left: 0;
+  max-height: 770px;
+  box-shadow: inset 0 0 100px 100px rgba(0, 0, 0);
```
Comment (Alekzann) [inline src/features/MoviePage/Banner/styled.js]: What do you think about small changing of box-shadow from this to 
box-shadow: inset 0px 0px **150px 140px** rgba(0, 0, 0);
At least for bigger displays.

**Review by Alekzann — APPROVED** [review]
Great job!

**Pull request closed without merging**